app/controllers/content.js

```diff
@@ -22,6 +22,12 @@ export default Ember.Controller.extend({
              this.set('authors', contributors));
     }),
 
+    applyMathJax: Ember.observer('model', function() {
+        Ember.run.next(this, function() {
```

Comment:
Can you please comment on the uses of `Ember.run` in this PR?

While sometimes necessary inside unit tests, this is usually not necessary within page code.